### PR TITLE
[Refactor] Add LABEL_ALREADY_EXISTS error for transaction stream load

### PR DESCRIPTION
**Pull request by meegoo**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 5 / 8 (62.50%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/rest/TransactionLoadAction.java|3|6|50.00%|[90, 91, 92]|
|:large_blue_circle:|com/starrocks/http/rest/ActionStatus.java|2|2|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/http/rest/ActionStatus.java [hunk 1]
```diff
@@ -20,5 +20,6 @@
 // Status of RESTful action
 public enum ActionStatus {
     OK,
-    FAILED
+    FAILED,
+    LABEL_ALREADY_EXISTS
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/http/rest/ActionStatus.java:24]: license header should be changed accordingly.

**Review by Astralidea — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Pull request merged**